examples/jsm/loaders/DRACOLoader.js

```diff
@@ -156,7 +157,12 @@ class DRACOLoader extends Loader {
 				} );
 
 			} )
-			.then( ( message ) => this._createGeometry( message.geometry ) );
+			.then( ( message ) => this._createGeometry( message.geometry ) )
+			.catch( ( error ) => {
+
+				throw new Error( error );
+
+			} );
```

Comment:
nit: I think the catch-and-then-throw block is not really doing anything, won't we have the same behavior with or without it? If there's a special purpose for this we may need to call it out in a comment.

Reply:
Good point. Yes, I think the relevant addition is the new `catch()` when calling `decodeGeometry()`. This indeed seems rendundant.

@Benjythebee Can you confirm?

Reply:
I apologize, I believe I did this for my own sanity while debugging this;
I can 100% send another PR to remove this portion of the code. Let me know!

Reply:
https://github.com/mrdoob/three.js/pull/27308 feel free to mark as PR and merge; Close if not good

Reply:
Thanks! New PR already merged!